Source/IntegrationTests/TestFiles/LitTests/LitTest/logger/CSVLogger.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %exits-with 4 %baredafny verify --log-format:csv";"LogFileName="%t.csv" "%s"
+// RUN: %exits-with 4 %baredafny verify --log-format:csv";"LogFileName="%t.csv" "%s" > %t
```

Comment:
If the file `%t` ever used for anything?

Reply:
I guess not :-D